core/templates/dev/head/pages/admin/activities_tab/admin_dev_mode_activities_tab_directive.html

```diff
@@ -71,3 +71,13 @@ <h3>Reload a single collection</h3>
     </div>
   </div>
 </md-card>
+
+<style>
+    .oppia-reload-collection-row {
```

Comment:
If both classes are going to have the same styling try using `,` or If possible find a common name.

Reply:
Done.